doc/guide/Guide_get_started.ipynb

```diff
@@ -0,0 +1,94 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "touched-penetration",
+   "metadata": {},
+   "source": [
+    "## Tutorials\n",
+    "A good way to start using CLIMADA is to have a look at the [Tutorials](https://github.com/CLIMADA-project/climada_python/tree/main/doc/tutorial). The [Main Tutorial](https://github.com/CLIMADA-project/climada_python/blob/main/doc/tutorial/1_main_climada.ipynb) will introduce you the structure of CLIMADA and how to calculate you first impacts, as well as your first appraisal of adaptation options. You can then look at the specific tutorials for each module (for example if you are interested in a specific hazard, like [Tropical Cyclones](https://github.com/CLIMADA-project/climada_python/blob/main/doc/tutorial/climada_hazard_TropCyclone.ipynb), or in learning to [estimate the value of asset exposure](https://github.com/CLIMADA-project/climada_python/blob/main/doc/tutorial/climada_entity_LitPop.ipynb),...). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "growing-religious",
+   "metadata": {},
+   "source": [
+    "## Contributing\n",
+    "If you would like to participate in the development of CLIMADA, have a look at the [Git and Development Guide](https://github.com/CLIMADA-project/climada_python/blob/main/doc/guide/Guide_Git_Development.ipynb). Before making a new feature, discuss with one of the repository admins. Every new feature or enhancement should be done on a separate branch, which will be merged in the develop branch after being reviewed (see [Checklist](https://github.com/CLIMADA-project/climada_python/blob/main/doc/guide/Guide_Reviewer_Checklist.ipynb)). Finally, the develop branch is merged in the main branch in each CLIMADA release. Each new feature should come with a tutorial and with [Unit and Integration Tests](https://github.com/CLIMADA-project/climada_python/blob/main/doc/guide/Guide_Continuous_Integration_and_Testing.ipynb). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "connected-anthony",
+   "metadata": {},
+   "source": [
+    "## Other Questions\n",
+    "\n",
+    "If you have any other questions, you might find some information in the [Miscellaneous guide](https://github.com/CLIMADA-project/climada_python/blob/main/doc/guide/Guide_Miscellaneous.ipynb). If you cannot find you answer in the guides, you can open an [issue](https://github.com/CLIMADA-project/climada_python/issues) for somebody to help you."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.8"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
```

Comment:
I added this guide with very concise information which links to all other guides

Reply:
Great point! Thanks a lot. 

We maybe should now coordinate with @emanuel-schmid so that the different guides are presented in good order on the documentation page. Maybe we should do some guide renaming too.

Reply:
The documentation page is created based on `doc/guide/index.rst` and, recursively, the documentation files listed there.\
The listed documentation files are stripped from their file extensions. Files with ipynb or rst extensions can be used. Setting up an order is done by making use of the markdown 'head level' (#, ## etc.) and the 'maxdepth' specification.

Once we have a design for the presentation on paper, we just have to edit the index.rst file until the result matches.

Right now it looks like 
```
----------
User guide
----------

.. toctree::
   :maxdepth: 2

   guide/introduction
   guide/install
   guide/tutorial
   guide/data_dependencies
   guide/configuration
   guide/developer
   guide/git_flow
...
```

I'd suggest to change that to something like:
```
----------
User guide
----------

.. toctree::
   :maxdepth: 2

   guide/introduction
   guide/install
   guide/data_dependencies
   
----------
Developer guide
----------

.. toctree::
   :maxdepth: 2

   guide/Guide_Configuration.ipnb
   guide/Guide_GitHub.ipnb
   ...
   guide/Guide_Miscellaneous.ipnb
...
```

Reply:
https://github.com/CLIMADA-project/climada_python/pull/181/commits/d349590a74fbb7f023e5d4c6e37841fac2094d6e